python/ray/dashboard/modules/healthz/healthz_head.py

```diff
@@ -33,9 +35,5 @@ async def health_check(self, req: Request) -> Response:
 
         return HTTPServiceUnavailable(reason="Health check failed")
 
-    async def run(self, server):
+    async def init(self):
         pass
```

Comment:
Will we have a module that implements a non-empty `init()`?

Reply:
Yes. For example, `EventHead` https://github.com/ray-project/ray/pull/50428/files#diff-4da7e8e823b529b1cf58a5b1ee8acfac42c6390789f084730589c974a9fb1267L203-R221